back/config/tenant_templates/e2etests_template.yml

```diff
@@ -8017,6 +8017,7 @@ models:
       action: budgeting
       permitted_by: groups
       permission_scope_ref: *verifiedPb
+      # Do we need a similar permission here for annotating_document ?
```

Comment:
@alexander-cit WDYT?

If we do need a similar example of such permissions in the `e2etests_template`, do you know how I should add it? (I have completely forgotten).
Edit the [e2etests-template.stg.citizenlab.co](http://e2etests-template.stg.citizenlab.co/) platfrom, re-build the templates (or just that one, maybe), then copy the file into this repo? (I'm really guessing)